src/os/bluestore/BlueRocksEnv.cc

```diff
@@ -195,11 +195,13 @@ class BlueRocksWritableFile : public rocksdb::WritableFile {
     return c_DefaultPageSize;
     }*/
 
+  using rocksdb::WritableFile::Append;
   rocksdb::Status Append(const rocksdb::Slice& data) override {
     fs->append_try_flush(h, data.data(), data.size());
     return rocksdb::Status::OK();
   }
 
+  using rocksdb::WritableFile::PositionedAppend;
```

Comment:
instead of exposing the `rocksdb::WritableFile::PositionedAppend()` from the base class, how about removing this override? as it is identical to the overridden one.